_includes/head.html

```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet" href="{{ "/assets/fonts/font-awesome-4.4.0/css/font-awesome.min.css" | prepend: site.baseurl }}">
   <link rel="stylesheet" href="{{ "/assets/main.css" | prepend: site.baseurl }}">
   <link rel="stylesheet" href="{{ "/assets/css/ekko-lightbox.css" | prepend: site.baseurl }}">
+  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
```

Comment:
Can you change this to href="{{ "favicon.ico" | prepend: site.baseurl }}"? Not including the site.baseurl leads to broken paths on GitHub Pages.